tensorboard/webapp/widgets/histogram/histogram_component.ng.html

```diff
@@ -74,18 +74,18 @@
     <g
       #histograms
       [class.histograms]="true"
-      [class.linked-time-enabled]="linkedTime"
-      [class.linked-time-single-step]="linkedTime && !linkedTime.end"
+      [class.linked-time-enabled]="timeSelection"
```

Comment:
Is it worth also updating linked-time-enabled to time-seleciton-enabled? And linked-time-single-step to time-selection-single-step? Up to you.

Reply:
Thanks for the catch! class .linked-time-enabled is no longer used. removed
also rename class name from `linked-time` to `histogram-card-fob` for element`<histogram-card-fob-controller`